index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">    
+    <link rel="stylesheet" href="./style.css">
+    <title>Conference Collaborative Software Development</title>
+</head>
+<body>
+    <div class="top-outline">
+        <ul class="outline-items">
+            <li><i class="fa fa-github"></i></li>
+            <li><i class="fa fa-facebook"></i></li>
+            <li><a href="#">Discover</a></li>
+            <li><a href="#">Login</a></li>
+            <li><a href="#">SignUp</a></li>
+            <li></li>
+        </ul>
+    </div>
+        <header class="topbar">
+            <nav class="navbar">
+                <ul class="nav-items">
+                    <li class="desktop-logo">
+                        <img class="logo-img" src="./images/bgimages/topLogo.png" alt="cc-log" >
+                        <span class="logo-text-home">CC 2022 <br> Collaborative Software Development</span>
+                    </li>
+
+                    <li class="li-deco"><a href="about.html">About</a></li>
+                    <li class="li-deco"><a href="#program">Program</a></li>
+                    <li class="li-deco"><a href="#join">Join</a></li>
+                    <li class="li-deco"><a href="#partners">Sponsor</a></li>
+                    <li class="li-deco"><a href="#">News</a></li> 
+                    <li class="li-deco key"><a href="#">CC campaign</a></li> 
+                </ul>
+                <div class="hamburger" id="hamburger-menu"><a href="#">&#9776;</a></div>
```

Comment:
- [ ]  Previous reviewers suggested to fix the issue of  wiered behavior of your navbar but seemed that you've deleted home nav, and its necessary on your page because I will not be able to navigate from about nav to home nav. so please fix it in right way 
![image](https://user-images.githubusercontent.com/43877689/181474119-556e1678-bc9b-4d0e-ad89-cf609bde50c5.png)